src/test/scala/chiselTests/AutoClonetypeSpec.scala

```diff
@@ -83,12 +83,10 @@ class RecordAutoCloneType[T <: Data](gen: T) extends Record with AutoCloneType {
 
 // Records that don't mixin AutoCloneType should still be able to implement the related methods
 // NOTE: This is a very weird thing to do, don't do it.
-class RecordWithVerbotenMethods(w: Int) extends Record {
+class RecordWithVerbotenMethods(w: Int) extends Record with AutoCloneType {
```

Comment:
See this comment, this test no longer applies with autoclonetype being mandatory, just delete it.